src/test/java/com/easypost/RefundTest.java

```diff
@@ -27,16 +27,19 @@ public class RefundTest {
      */
     @BeforeAll
     public static void setup() throws EasyPostException {
-        _vcr = new TestUtils.VCR("refund", TestUtils.ApiKey.TEST);
+        vcr = new TestUtils.VCR("refund", TestUtils.ApiKey.TEST);
     }
 
     /**
      * Create a list of refunds.
+     *
+     * @return List of Refund objects
      */
     private static List<Refund> createBasicRefundList() throws EasyPostException {
         Shipment shipment = Shipment.create(Fixture.oneCallBuyShipment());
 
-        Shipment retrievedShipment = Shipment.retrieve(shipment.getId()); // We need to retrieve the shipment so that the tracking_code has time to populate
+        Shipment retrievedShipment = Shipment.retrieve(shipment.getId());
+        // We need to retrieve the shipment so that the tracking_code has time to populate
```

Comment:
Comments associated with a single line of code should occur before the line in question. Doing so afterwords is confusing and misleading to the intent of the comment. Since Checkstyle won't let use do this inline due to line length (should be 120 chars right?), then let's have this comment come before line 41.